src/dso_api/sentry.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, Final, Optional
+from urllib.parse import urlparse
+
+import environ
+
+env = environ.Env()
+
+# The makers of sentry only have internal types (in `_types`)
+# those are not considered stable atm, so we define
+# our own aliasses for now.
+Event = dict[str, Any]
+Hint = dict[str, Any]
+
+
+SENTRY_BLOCKED_PATHS: Final[str] = env.str("SENTRY_BLOCKED_PATHS", "")
```

Comment:
`env.list()` gebruiken en dan wordt het al opgeplitst vanuit comma-separated

Reply:
Deze variabele kan je wel gewoon in de settings file houden. Dan werken alle logica (zoals `@override_settings` ook gewoon).

Reply:
Ah, idd, handig!